dataactcore/scripts/pullFPDSData.py

```diff
@@ -2259,6 +2261,16 @@ def create_unique_key(row):
     return unique_string
 
 
+def create_unique_award_key(row):
+    key_list = ['piid', 'agencyid', 'idvpiid', 'idvagencyid'] if row['pulled_from'] == 'award' else ['piid', 'agencyid']
+    unique_string_list = [] if row['pulled_from'] == 'award' else [row['pulled_from']]
+
+    for item in key_list:
+        unique_string_list.append(row[item] if row[item] and str(row[item]) != 'nan' else '-none-')
+
+    return '_'.join(unique_string_list)
```

Comment:
This is only handling the key for historical data. What about for new data coming in?